package.json

```diff
@@ -1070,10 +1070,10 @@
                     "default": 8081,
                     "description": "Port to use when connecting to a CosmosDB Emulator instance"
                 },
-                "azureDatabases.testCosmosAuth": {
+                "azureDatabases.useCosmosOAuth": {
```

Comment:
My testing indicates this setting is still needed. The built-in Cosmos DB roles both allow people to list keys even if they cannot use the key due to disabling local auth. This setting allows people to prefer OAuth credential over the listed key.